storybookjs/storybook
# Addon-docs: Vue SFC support in MDX

Issue: https://github.com/storybookjs/storybook/issues/9768

## What I did

I added a transformer for Vue stories written in SFC syntax:

``````mdx
<Story name="to storybook" height="300px">
  ```vue
  <template>
    <button>click me {{ name }}</button>
  </template>
  <script>
    export default {
      data() {
        return {
          name: 'foo'
        }
      }
    }
  </script>
  ```
</Story>
``````

## How to test

- Is this testable with Jest or Chromatic screenshots?
Yes, I added two snapshot tests for stories written in SFC syntax.

- Does this need a new example in the kitchen sink apps?
Yes, I added an example in the vue kitchen sink (Addon > Docs > SFC Syntax).

- Does this need an update to the documentation?
No / I cannot find the right place to put it.

If your answer is yes to any of these, please make sure to include it in your PR.
